add-on/src/popup/quick-upload.js

```diff
@@ -62,14 +79,41 @@ function quickUploadStore (state, emitter) {
   })
 }
 
+function quickUploadOptions (state, emit) {
+  const onExpandOptions = (e) => { state.expandOptions = true; emit('render') }
+  const onWrapWithDirectoryChange = (e) => { state.wrapWithDirectory = e.target.checked }
+  const onPinUploadChange = (e) => { state.pinUpload = e.target.checked }
+  if (state.expandOptions) {
+    return html`
+      <div id='quickUploadOptions' class='sans-serif mt3 f6 lh-copy light-gray no-user-select'>
+        <label for='wrapWithDirectory' class='flex items-center db relative mt1 pointer'>
+          <input id='wrapWithDirectory' type='checkbox' onchange=${onWrapWithDirectoryChange} checked=${state.wrapWithDirectory} />
+          <span class='mark db flex items-center relative mr2 br2'></span>
+          ${browser.i18n.getMessage('quickUpload_options_wrapWithDirectory')}
+        </label>
+        <label for='pinUpload' class='flex items-center db relative mt1 pointer'>
+          <input id='pinUpload' type='checkbox' onchange=${onPinUploadChange} checked=${state.pinUpload} />
+          <span class='mark db flex items-center relative mr2 br2'></span>
+          ${browser.i18n.getMessage('quickUpload_options_pinUpload')}
+        </label>
+      </div>
+    `
+  }
+  return html`
+    <button class='f6 link dim bn bg-transparent moon-gray dib pa0 pointer ma0' style='color: #6ACAD1' onclick=${onExpandOptions}>
```

Comment:
Add an `mt3` here to get the same vertical space when the button transitions to the options.